static/src/javascripts/projects/common/modules/commercial/fiv-banner.js

```diff
@@ -27,7 +27,7 @@ const hideBanner = () => {
 };
 
 const defaultEngagementBannerParams = () => ({
-    campaignCode: 'gdnwb_copts_fiv_banner',
+    campaignCode: 'onemillion',
```

Comment:
this is the code being used overall to group together all the tracking.  The component tracking below is still more unique names for this banner.